docs/tutorials/tutorials-mining-tokens-topup-supernodes.md

```diff
@@ -0,0 +1,89 @@
+---
+id: tutorials-mining-tokens-topup-supernodes
+title: Topup Supernodes
+sidebar_label: Topup Supernodes
+---
+
+## How the community may propose to transfer DHX tokens from the DHX DAO Unlocked Reserves (Treasury) to a Multisig Proxy account for subsequent transfer to topup a Supernode
+
+### Step 1: Check the DataHighway runtime parameters are reasonable values and modify them if necessary
+
+**Important:** It may be necessary to use the democracy runtime module to propose to modify the current runtime configuration so that the Minimum Deposit, Launch Period, and Voting Period are reasonable values. If they were set with unreasonable values at genesis, then it may be necessary to modify them using Sudo. 
+
+### Step 2: Generate a Preimage Hash of the Proposal to transfer Treasury funds to the Multisig Proxy account
+
+* Add the Multisig Proxy account and the DHX DAO Unlocked Reserves (Treasury) account (4LTFqiD6H6g8a7ur9WH4RxhWx2givWfK7o5EDed3ai1nYTvk) to your [Address book](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet-harbour.datahighway.com#/addresses)
+	* Note: The Treasury account for each chain is specified in the [chain_spec.rs](https://github.com/DataHighway-DHX/node/blob/master/node/src/chain_spec.rs)
+
+* Go to _Democracy_ page https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet-harbour.datahighway.com#/democracy
+
+* The first step is to submit a preimage of the proposal.
+
+* Click on _Submit preimage_
+
+* Choose for _send_from_account_ an account (with sufficient balance to pay any fees)
+
+* Choose for _propose_ an extrinsic _balances_ and select _forceTransfer_ as function
+
+* Choose for _source_ the account of the Treasury (DHX DAO Unlocked Reserves).
+
+* Choose for _dest_ the account of the Multisig Proxy.
+
+* Choose for _value_ the value in DHX that you propose to be transferred to the Multisig Proxy that will perform the subsequent transfer to the Supernode account
+
+* Copy the preimage hash. This represents the proposal. You will use this hash when submitting the actual proposal.
+
+### Step 3: Submit the Proposal
+
+* Select the account from which you want to submit the proposal (with sufficient balance to pay any fees)
+
+* Enter the preimage hash related to the proposal (copied in the previous step)
+
+* Set the locked balance. 
+
+* Click the _Submit proposal_ button and sign the transaction
+
+### Step 4: Submit the Preimage Hash of the Proposal to transfer treasury funds to the Multisig Proxy account
```

Comment:
@ltfschoen Could you please revalidate this step? As I checked, this step won't occur. Below steps would happen as a part of Step 2

* Leave the value for the _weight parameter at the default of 0.

* Click "Submit Transaction" and then "Sign and Submit".


After submitting the proposal, it will go for `Seconding the Proposal`, which you already mentioned in Step 5.

Reply:
Yes, the order appears to be weird, but that's how I got it to work for proposal no. 1 here https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet-harbour.datahighway.com#/democracy

It's also in the same order they say to do it here https://wiki.polkadot.network/docs/maintain-guides-democracy/#proposing-an-action

Reply:
The reason why proposal no. 1 is taking so long to become tabled as a referendum for voting is because its configured to be a really long time by default in the runtime. 
So I think we'd have no choice but to perform a runtime upgrade using Sudo (instead of democracy) to change the Minimum Deposit, Launch Period, and Voting Period

Reply:
This document https://wiki.polkadot.network/docs/maintain-guides-democracy/#proposing-an-action needs to be corrected. 

We as a developer can understand since we do experiments 😊  But End-user will get confused. So we need to write steps in user-friendly language.

Reply:
yes i was questioning their approach too and i agree your approach is much simpler. i've just tested it fully and it achieves the same outcome so we'll do that.

Reply:
i've pushed a commit addressing your comments